### ISSUE TITLE
Perhaps unintended cooperation between geometric distortion correction and Lens/Geometry in Transform-tab?

### ISSUE THREAD

**Issue by hholtm** [Beep6581/RawTherapee#5826]
If in Transform-tab in Profiled Lens Correction Geometric distortion is activated and in Lens/Geometry Auto-fill is activated too I get an unintended crop although everything else inside Lens/Geometry is untouched. If Auto-fill is deactivated, everything works fine and I get no unintended crop.
This behaviour seems to be relatively new, perhaps it came with new functions inside Lens/Geometry?
@Lawrence37 : Can you check if there is an unintended cooperation between these functions followed by a crop or, if this behaviour is intended, if the Auto-fill button should get a hint which indicates this behaviour?
I am working with RT-dev 5.8-2276 on Linux Mint 19.3. Perhaps someone can test, if the above mentioned behaviour can be confirmed.

**Comment by Lawrence37** [comment]
I might have found the problem. Let me see if I can fix it.

**Comment by Lawrence37** [comment]
Thanks for reporting the bug. Are you able to test the change I made?

**Comment by Thanatomanic** [comment]
Hi @Lawrence37 quick note: for such a simple fix a new branch and a PR is a bit overkill imo. You could use `git diff` and provide a patch here for people to try out. For an example: #5564

**Comment by hholtm** [comment]
@Lawrence37  I'm using AppImages. So I can test only when the fix is integrated in an AppImage.

**Comment by Lawrence37** [comment]
The problem comes from a change I neglected to make in one of my pull requests. It's a very small change. Should I commit directly to dev then? Here's the diff for reference.

```diff
diff --git a/rtengine/iptransform.cc b/rtengine/iptransform.cc
index 292d96383..8a1142c17 100644
--- a/rtengine/iptransform.cc
+++ b/rtengine/iptransform.cc
@@ -516,7 +516,7 @@ bool ImProcFunctions::transCoord (int W, int H, const std::vector<Coord2D> &src,
         }
 
         if (pLCPMap && params->lensProf.useDist) {
-            pLCPMap->correctDistortion(x_d, y_d, 0, 0);
+            pLCPMap->correctDistortion(x_d, y_d, w2, h2);
         }
 
         // rotate
```

**Comment by casta** [comment]
Great thank’s ! I noticed the issue since some time on master. This change fixes the issue. Hope it will be soon pushed.

**Comment by Lawrence37** [comment]
@hholtm The fix is in dev and will be available in version 5.8-2309 or higher.
